src/NServiceBus.Core/Audit/Audit.cs

```diff
@@ -10,7 +10,11 @@ public class Audit : Feature
         internal Audit()
         {
             EnableByDefault();
-            Prerequisite(config => AuditConfigReader.GetConfiguredAuditQueue(config.Settings, out auditConfig), "No configured audit queue was found");
+            Defaults(settings =>
+            {
+                settings.Set<AuditConfigReader.Result>(AuditConfigReader.GetConfiguredAuditQueue(settings));
```

Comment:
I don't think there's an issue. Both defaults and prerequisites are checked in the call chain invoked by `FeatureActivator.SetupFeatures`.

IProvideConfiguration sources are searched in the scanned types as far as I saw, so there is no change to those in between as far as I can tell.